common/cme_api_postman/v1/cme_api_v1.2.postman_collection.JSON

```diff
@@ -0,0 +1,1256 @@
+							"host": [
+								"{{server}}"
+							],
+							"path": [
+								"v1.8",
+								"cme-api",
+								"v1.2",
+								"accounts",
+								"gcp"
+							]
+						}
+					},
+					"response": []
+				},
+				{
+					"name": "PUT set GCP account",
+					"request": {
+						"method": "PUT",
+						"header": [
+							{
+								"key": "Content-Type",
+								"value": "application/json"
+							},
+							{
+								"key": "X-chkp-sid",
+								"value": "{{session}}"
+							}
+						],
+						"body": {
+							"mode": "raw",
+							"raw": "{\"project_id\": \"<GCP_project_id>\", \"credentials_file\": \"GCP_credentials_file_name\", \"deletion_tolerance\": \"<deletion_tolerance>\"}",
+							"options": {
+								"raw": {
+									"language": "json"
+								}
+							}
+						},
+						"url": {
+							"raw": "{{server}}/v1.8/cme-api/v1.2/accounts/gcp/<account_name>",
+							"host": [
+								"{{server}}"
+							],
+							"path": [
+								"v1.8",
+								"cme-api",
+								"v1.2",
+								"accounts",
+								"gcp",
+								"<account_name>"
+							]
+						}
+					},
+					"response": []
+				},
+				{
+					"name": "POST add GCP gw configuration",
+					"request": {
+						"method": "POST",
+						"header": [
+							{
+								"key": "X-chkp-sid",
+								"value": "{{session}}",
+								"type": "text"
+							},
+							{
+								"key": "Content-Type",
+								"value": "application/json",
+								"type": "text"
+							}
+						],
+						"body": {
+							"mode": "raw",
+							"raw": "{\"name\": \"<gwConfigurations_name>\", \"base64_sic_key\": \"<encoded_base64_sic_key>\", \"version\": \"<gw_version>\", \"policy\": \"<policy_name>\", \"blades\": {\"ips\":true, \"anti-bot\":true, \"url-filtering\": true, \"https-inspection\": true}, \"related_account\": \"<associated_account>\"}"
+						},
+						"url": {
+							"raw": "{{server}}/v1.8/cme-api/v1.2/gwConfigurations/gcp",
+							"host": [
+								"{{server}}"
+							],
+							"path": [
+								"v1.8",
+								"cme-api",
+								"v1.2",
+								"gwConfigurations",
+								"gcp"
+							]
+						}
+					},
+					"response": []
+				},
+				{
+					"name": "PUT set GCP gw configuration",
+					"request": {
+						"method": "PUT",
+						"header": [
+							{
+								"key": "X-chkp-sid",
+								"value": "{{session}}",
+								"type": "text"
+							},
+							{
+								"key": "Content-Type",
+								"value": "application/json",
+								"type": "text"
+							}
+						],
+						"body": {
+							"mode": "raw",
+							"raw": "{\"base64_sic_key\": \"<encoded_base64_sic_key>\", \"version\": \"<gw_version>\", \"policy\": \"<policy_name>\", \"blades\": {\"ips\":true, \"anti-bot\":true, \"url-filtering\": true, \"https-inspection\": true}, \"related_account\": \"<associated_account>\"}"
+						},
+						"url": {
+							"raw": "{{server}}/v1.8/cme-api/v1.2/gwConfigurations/gcp/<gwConfigurations_name>",

```

Comment:
Please change to "raw": "{{server}}/v1.8/cme-api/v1.2/gwConfigurations/gcp/<**gwConfiguration_name**>",